src/types/config.d.ts

```diff
@@ -28,5 +29,14 @@ export interface LxdConfigOptions {
     instance: LxcConfigOptionCategories;
     project: LxcConfigOptionCategories;
     server: LxcConfigOptionCategories;
+    "storage-btrfs": LxcConfigOptionCategories;
+    "storage-ceph": LxcConfigOptionCategories;
+    "storage-cephfs": LxcConfigOptionCategories;
```

Comment:
do we want to include `storage-cephfs` and `storage-cephobject` in this type now even if it's not supported in the ui yet?

Reply:
Types prefixed with `Lxd` usually relate to things exposed by the LXD api. So this represents what the api has to offer. I think it's okay to keep it here even though we don't use it yet.